scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -175,10 +170,6 @@ function Initialize-IoTEdge {
         [ValidateNotNullOrEmpty()]
         [String] $X509IdentityPrivateKey,
 
-        # Auto generate the X.509 identity certificate from the device CA
```

Comment:
Why is this being removed? The commit message doesn't say.

This is in 1.0.8. What about users who are already using it?

Reply:
This mode of operation was never released so there is no compatibility concern here.

Reply:
Also I realized, there is other cleanup in the HSM layer as well that is needed to fully remove this. I will create a separate PR and tag it here. I don't want to clutter this PR with those changes.

Reply:
https://github.com/Azure/iotedge/pull/1495